src/io/flutter/editor/WidgetIndentsHighlightingPassFactory.java

```diff
@@ -49,7 +49,7 @@
   // widget indent guide lines after editing code containing guides.
   private static final boolean SIMULATE_SLOW_ANALYSIS_UPDATES = false;
 
-  private final Project project;
+  private Project project;
```

Comment:
Otherwise the Java type system gets cranky about accessing project from settingsListener.